test/controllers/MultiCacheControllerTest.ts

```diff
@@ -35,6 +38,8 @@ describe('/v1/caches', () => {
           .get('/zones/purge/1.json');
         fasterizeFlushMock = fasterizeAPIMock
           .delete('/v1/configs/42/cache');
+        fastlyFlushMock = fastlyAPIMock
+          .post('/service/abcd/purge_all');
```

Comment:
The mock for Fastly API is setup but no test use it thus there is no test that assert the code added here https://github.com/fasterize/darkstar/pull/4/commits/147bf6837af06224d8bde273a688e5ef9afa6d52#diff-0311da4455cc94ee09a0b49aac70e67eR62 is functional

Reply:
@dax fixed the test case